### PR TITLE
Add resources for Ping, Availability alerts, and Monitor action group

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: monitor-action-group-email/main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+resource "null_resource" "dependency_getter" {
+  provisioner "local-exec" {
+    command = "echo ${length(var.dependencies)}"
+  }
+}
+resource "null_resource" "dependency_setter" {
+  depends_on = [azurerm_monitor_action_group.main]
```
Comment (Renetnielsen) [inline monitor-action-group-email/main.tf:20]: Indentation
```
depends_on = [
  azurerm_monitor_action_group.main
]
```

File: monitor-action-group-email/main.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+resource "azurerm_monitor_action_group" "main" {
+  depends_on             = [null_resource.dependency_getter]
+  name                   = var.name
+  resource_group_name    = var.resource_group_name 
+  short_name			 = var.short_name
```
Comment (Renetnielsen) [inline monitor-action-group-email/main.tf:27]: Indentation

**Review by Renetnielsen — CHANGES_REQUESTED** [review]
remove the .gitignore file. Modules will not be run inside this repository. Therefor not creating any unwanted folders.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,400 @@
+﻿# Copyright 2020 Energinet DataHub A/S
```
[the remark this answers is not in the document]
Comment (djorgensendk) [reply .gitignore:1]: shorten as must as possible

File: availability-alert/main.tf [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  criteria {
+    metric_namespace       = "Microsoft.Insights/components"
+    metric_name            = "availabilityResults/count"
+    aggregation            = "Count"
+    operator               = "GreaterThan"
+    threshold              = var.threshold
+    dimension {
+      name                 = "availabilityResult/name"
+      operator             = "Include"
+      values               = [ var.ping_test_name ]
```
[the remark this answers is not in the document]
Comment (djorgensendk) [reply availability-alert/main.tf]: fixed

File: availability-alert/main.tf [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    }
+	dimension {
+      name                 = "availabilityResult/success"
+      operator             = "Include"
+      values               = [ "0" ]
```
[the remark this answers is not in the document]
Comment (djorgensendk) [reply availability-alert/main.tf]: fixed

File: monitor-action-group-email/main.tf (hunk 1 above)
Comment (djorgensendk) [reply monitor-action-group-email/main.tf]: fixed

File: monitor-action-group-email/main.tf (hunk 2 above)
Comment (djorgensendk) [reply monitor-action-group-email/main.tf]: fixed

**Pull request merged**